docs/source/pillars/information_governance.md

```diff
@@ -156,64 +295,200 @@ _The ability of the TRE organisation to manage projects effectively._
+:header-rows: 1
+:name: tab-ig-user-onboarding
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have clear onboarding processes in place for all roles within your TRE organisation.
+  - This may include all members signing role-specific terms of use and completing role specific training.
+  - Mandatory
+```
+
+**Identity and Access Management Services:** Govern and control user identities, access privileges, authentication, and authorization within an organisation.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-training-management-delivery
+:name: tab-ig-identity-access-management-services
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have a set of services to manage access to resources based on identity.
+  - This will include a security model for role based access with technical controls to ensure the principle of least privilege is enforced.
+  - Mandatory
+* - The data asset owner must have control of who has access to which assets.
+  - Either through policy and process or ideally direct control access to data should be controlled by the person who owns a data asset or their delegates.
+  - Mandatory
+```
+
+**Authentication Application:** A software system that verifies and validates the identities of users or entities accessing a system through multifactor authentication.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-authentication-application
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have robust and secure applications in place to authenticate users (and services) within the TRE.
+  - The number of authentication applications should be kept to a minimum with common controls and standards applied across all such as MFA, password complexity etc.
+  - Mandatory
+```
+
+**User Identity Attributes:** Characteristics or attributes associated with a user's identity, such as username, email address, role, permissions, or affiliations.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-user-identity-attributes
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must give each user of the TRE a unique logon with changes to any records strictly controlled.
+  - The unique identifier and all associated records for a user should be traceable across the entire TRE. This will include training records, affiliations, contract agreements and ethics approvals where required.
+  - Mandatory
+```
+
+## Training Delivery and Management
+
+Ability to deliver, track and maintain adequate training levels to ensure competence of all people within the TRE organisation.
+
+**Curriculum Creation and Management:** The process of designing, developing, and managing educational curricula, courses through training needs analysis for required competency.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-curriculum-creation-management
 
 * - Statement
   - Guidance
   - Importance
 * - You must have relevant training for all roles within the TRE organisation, and the ability to deliver this training.
```

Comment:
Why should "you" (the TRE operator) have to deliver the training? Why not outsource or use a common accreditation (eg. ONS Safe Researcher).

Reply:
I assume "you" means "you're responsible for", but are free to delegate it.

Reply:
```suggestion
* - You must ensure that relevant training is available for all roles within the TRE organisation.
```